SCClassLibrary/Common/Quarks/QuarksGui.sc

```diff
@@ -29,64 +31,80 @@ QuarksGui {
 		bounds = Window.flipY(Window.availableBounds);
 		window = Window("Quarks", Rect(0, 0, min(bounds.width * 0.75, 1000), bounds.height * 0.9).center_(bounds.center));
 
-		lblCaption = StaticText().font_(GUI.font.new(size:16, usePointSize:true)).string_("Quarks");
+		colors = (
+			error: (bg: Color.fromHexString("#fcdede"), text: Color.fromHexString("#991111")),
+			info: (bg: Color.fromHexString("#e2eef9"), text: Color.fromHexString("#224466")),
+			warn: (bg: Color.fromHexString("#f7ea57"), text: Color.fromHexString("#494620")),
+			success: (bg: Color.fromHexString("#f5ffef"), text: Color.fromHexString("#6cc644")),
+			primary: (bg: Color.fromHexString("#76c656"), text: Color.white),
+			installed: (bg: Color.fromHexString("#76c656"), text: Color.white),
+			default: (bg: Color.white, text: Color.black)
+		);
 
 		btnUpdateDirectory = Button()
-			.states_([["Refresh Quarks directory"]])
-			.toolTip_("Download directory listing from" + Quarks.directoryUrl + "(auto-refreshes every 4 hours)")
+			.states_([["Check for updates"], ["Checking..."]])
```

Comment:
Previously this just fetched the directory file. Now it also fetches each downloaded quark to get any new tags so it can show that there are new versions available to install.